portfolio/src/main/webapp/script.js

```diff
@@ -471,6 +472,8 @@ function scrollTerminalToBottom() {
 
 function openTerminal() {
   document.getElementById('terminal').style.display = 'initial';
+  // Timeout because I have to wait the terminal to render
```

Comment:
I'd recommend using passive language, e.g. "Timeout to wait for the terminal to render" rather than using I/we

Reply:
Right, seems better. Thanks.